lib/dependency_spy/cli.rb

```diff
@@ -46,8 +49,9 @@ class CLI < Thor
     method_option('output-path', :aliases => :o, :type => :string)
     method_option('database-path', :type => :string, :aliases => :p, :default => YAVDB::Constants::DEFAULT_YAVDB_DATABASE_PATH)
     method_option('offline', :type => :boolean, :default => false)
-
+    method_option('severity-threshold', :aliases => :s, :type => :string, :enum => SEVERITY_OPTIONS, :default => 'low')
     def check
+      severity_threshold = options['severity-threshold'] || 'low'
```

Comment:
Since you have a default, why do you need this `||`?

Reply:
Done.